code/game/machinery/adv_med.dm

```diff
@@ -410,7 +410,7 @@
 			visible_message("<span class='notice'>\The [src] rattles and prints out a sheet of paper.</span>")
 			var/obj/item/weapon/paper/P = new /obj/item/weapon/paper(loc)
 			P.info = "<CENTER><B>Body Scan - [href_list["name"]]</B></CENTER><BR>"
-			P.info += "<b>Time of scan:</b> [worldtime2text(world.time)]<br><br>"
+			P.info += "<b>Time of scan:</b> [stationtime2text(world.time)]<br><br>"
```

Comment:
Btw, what is that addition to the exception catching proc? In `scheduler.dm`
at line 329.
On Sat, Jul 30, 2016, 19:25 PsiOmegaDelta <notifications@github.com> wrote:

> In code/game/machinery/adv_med.dm
> <https://github.com/PolarisSS13/Polaris/pull/2232#discussion_r72886690>:
>
> > @@ -410,7 +410,7 @@
> >  			visible_message("<span class='notice'>\The [src] rattles and prints out a sheet of paper.</span>")
> >  			var/obj/item/weapon/paper/P = new /obj/item/weapon/paper(loc)
> >  			P.info = "<CENTER><B>Body Scan - [href_list["name"]]</B></CENTER><BR>"
> > -			P.info += "<b>Time of scan:</b> [worldtime2text(world.time)]<br><br>"
> > +			P.info += "<b>Time of scan:</b> [stationtime2text(world.time)]<br><br>"
>
> #13844 has the necessary time changes to make autopsy results report
> correctly (as well as fingerprints, attack logs, and other things).
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/PolarisSS13/Polaris/pull/2232/files/4f7ea573c15445ee667b9896c6f2e6c0b08a1d81#r72886690>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEFMuMJxQQPOn1kNpi1hGnhVCMDKEUkBks5qa0LCgaJpZM4JYwAa>
> .
>